habushu-maven-plugin/src/main/resources/dockerfile_final_stage_template

```diff
@@ -0,0 +1,5 @@
+RUN ln -sf $(which python) /venv/bin/python
```

Comment:
I would think we want to run this at the end of this template. Not sure it matters, but would rather copy `/venv/bin/python` into the final image, then update it's sym link

Reply:
Also would be helpful to have a comment with it, something like:
```
# Update the venv python symlink to point to the underlying python executable
```